learningresources/models.py

```diff
@@ -19,6 +19,16 @@
 
 log = logging.getLogger(__name__)
 
+# defining the file path max length
+# mac and ubuntu have a max path size of 1024, but leaving some room
```

Comment:
You should say linux instead of just ubuntu, and Linux has a max path of 4096 according to [here](http://unix.stackexchange.com/questions/32795/what-is-the-maximum-allowed-filename-and-folder-size-with-ecryptfs). No problem with 900 as a length though, just reword the comment for accuracy